resources/views/settings/tags.blade.php

```diff
@@ -72,6 +76,10 @@
                     </ul>
                   </div>
                   <div class="table-cell actions">
+                    <a href="javascript:;" class="edit-icon-btn" onclick="toggleEditInput(this)">
```

Comment:
```suggestion
                    <div class="edit-icon-btn" onclick="toggleEditInput(this)">
```
Will it work? In this way we can avoid `href="javascript:;"`.